### ISSUE TITLE
Explain in the FAQ what "Take me over?" means and how to get rid of it

### ISSUE THREAD

**Issue by szabgab** [metacpan/metacpan-web#2431]
When someone visits a CPAN module that is marked by ADOPTME or HANDOFF or NEEDHELP  they will see a banner with the tittle "Take me over?".

As a person who does not know much about Perl and CPAN I'd probably want to see some explanation what does that mean.

As a person who has experience in Perl and is interested to take over the distribution, but is not experienced in the working in PAUSE I might need a lot more explanation than the link to emails.

As a person who has just adopted a module I'd like to know how to remove that banner.

It would be nice to add an  explanation of  ADOPTME,  HANDOFF, and NEEDHELP  to the FAQ and link to it from the banner.

See also #1643

**Comment by bigpresh** [comment]
Looks good, this should help clarify it.  I also think maybe the notification banner on the module page should have a brief explanation (possibly just as a little "i" info icon that links to it, so it doesn't get too wordy) - possibly just linking to the new content in the FAQ here.

**Comment by oalders** [comment]
Closed via #2432
